resolve/RandSketchingCountSketch.cpp

```diff
@@ -47,6 +68,16 @@ namespace ReSolve
   }
 
   // Setup the parameters, sampling matrices, permuations, etc
+  /** 
+   * @brief Sketching method setup. This function allocated _labels_, and _flip_ arrays and, populates them.
+   *
```

Comment:
Make the brief really brief:
```suggestion
  /** 
   * @brief Sketching method setup.
   * 
   * This function allocated _labels_, and _flip_ arrays and, populates them.
   *
```
The empty line after the `@brief` will place the rest of the text in the long function description.